space-concordia-robotics/robotics-prototype
# Updates requirements file to get rid of version dependency errors

# Assignee Section

## Description
Update attrs and six to newer versions.

### Steps for Testing
1. Delete venv folder with `rm -rf`
2. Create new venv: ``virtualenv -p `which python3.6` venv``
3. activate venv: `source venv/bin/activate`
4. run `pip install -r requirements.txt -r requirements-dev.txt --no-cache-dir`
5. run setup: `python setup.py develop`
6. run `pytest` and verify it passes

closes #405 
closes #411 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
